### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by allencloud**
fix typos

### CODE REVIEW

File: runconfig/opts/parse.go [hunk 1]
```diff
@@ -246,7 +246,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*container.Config, *container.Host
 	// Validate if the given hostname is RFC 1123 (https://tools.ietf.org/html/rfc1123) compliant.
 	hostname := *flHostname
 	if hostname != "" {
-		// Linux hostname is limited to HOST_NAME_MAX=64, not not including the terminating null byte.
+		// Linux hostname is limited to HOST_NAME_MAX=64, not to include the terminating null byte.
```
Comment (tiborvass) [inline runconfig/opts/parse.go]: i think the problem was just a duplication of `not`, so this should just be `not including`

**Comment by thaJeztah** [comment]
LGTM

File: runconfig/opts/parse.go [hunk 2]
```diff
@@ -473,7 +473,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*container.Config, *container.Host
 	return config, hostConfig, networkingConfig, cmd, nil
 }
 
-// reads a file of line terminated key=value pairs and override that with override parameter
+// reads a file of line terminated key=value pairs and overrides that with override parameter
```
Comment (aaronlehmann) [inline runconfig/opts/parse.go]: "reads a file of line terminated key=value pairs, and overrides any keys present in the file with additional pairs specified in the override parameter"?

Comment (allencloud) [inline runconfig/opts/parse.go]: Thanks @aaronlehmann

**Comment by aaronlehmann** [comment]
LGTM

**Pull request merged**